CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## TBD
 ### Fixed
+* Show Functions now shows UDF's from all metastores. See [#164](https://github.com/HotelsDotCom/waggle-dance/issues/164).
```

Comment:
-> "`show functions` now shows UDFs" 
(no apostrophe for plurals unless it's for single letters or single numbers - I only know this because I googled it 😄)

Reply:
yeah I thought so but it looked weirder :) I'll change it :)